addons/im_livechat/tests/test_get_discuss_channel.py

```diff
@@ -346,3 +346,38 @@ def test_livechat_manager_can_invite_anyone(self):
             partner_ids=self.operators[0].partner_id.ids
         )
         self.assertEqual(self_member.partner_id, self.operators[0].partner_id)
+
+    def test_livechat_operator_can_see_all_livechat_conversations(self):
+        bob_user = new_test_user(
+            self.env, "bob_user", groups="base.group_user,im_livechat.im_livechat_group_user"
+        )
+        live_chat = self.env["discuss.channel"].create(
```

Comment:
This would be more appropriately named as discuss_channel or livechat_session (same in the other test).